shared/js/settings.js

```diff
@@ -7,16 +7,17 @@ var load = require('load');
  */
 require.scopes.settings =(() => {
     var settings = {};
+    // external settings defines a function that needs to run when a setting is updated
+    var externalSettings = {
+        'httpsEverywhereEnabled': function(value){ isExtensionEnabled = value }
+    };
```

Comment:
moved this above init so that it exists before it's attempted to be used in `buildSettingsFromLocalStorage`, since safari is sync it was causing an error.